README.md

```diff
@@ -51,7 +51,7 @@ use Trulyao\PhpRouter\Router as Router;
 require(__DIR__ . '/../vendor/autoload.php');
 
 # Create a new instance
-$router = new Router(__DIR__."/views", "examples");
+$router = new Router(__DIR__."/src", "examples");
```

Comment:
The `src` folder is not where the view is at, this assumes they have autoloader mapped to /src too and the index.php is outside that, the initial version was written to match the `examples` folder. Check it out for reference.

Reply:
Ok